app/build.gradle.kts

```diff
@@ -13,3 +13,14 @@ dependencies {
     implementation("org.springframework.data:spring-data-elasticsearch:_")
     implementation("org.springframework:spring-web:_")
 }
+
+project.configurations {
```

Comment:
Can you clarify why is this change required ?

Reply:
this was added because the fat jar was not getting created for the App module